lib/engine.py

```diff
@@ -477,8 +477,11 @@ def to_csv(self):
         # due to Cyclic imports we can not move this import to the top
         from retriever.lib.tools import sort_csv
         csvfile_output = (self.table_name() + '.csv')
-        csv_out = open(csvfile_output, "wb")
-        csv_writer = csv.writer(csv_out, dialect='excel')
+        csv_out = open(csvfile_output, "w")
+        if os.name == 'nt':
+            csv_writer = csv.writer(csv_out, dialect='excel', lineterminator='\n')
+        else:
```

Comment:
I think the `if else` part can go away and we just use c`sv_writer = csv.writer(csv_out, dialect='excel', lineterminator='\n')`